cpp/oneapi/dal/backend/interop/table_conversion.hpp

```diff
@@ -122,12 +123,147 @@ inline daal::data_management::NumericTablePtr convert_to_daal_table(const homoge
     return copy_to_daal_homogen_table<Data>(table);
 }
 
+template <typename T>
+inline auto convert_to_daal_csr_table(array<T>& data,
```

Comment:
merge with homogen as example: 
convert_to_daal_table -> convert_to_daal_homogen_table, convert_to_daal_csr_table

convert_from_daal_table -> convert_from_daal_homogen_table, convert_from_daal_csr_table

Reply:
Ok, it was fixed